src/ticTacToeGame.java

```diff
@@ -16,15 +17,72 @@ public class ticTacToeGame {
 
     public static void main(String[] args) {
         // TODO Auto-generated method stub
-
-        symbol_choosing();
+        boolean gameStarted = checkIfGameExists();
+        if (!gameStarted) {
+            loadGame();
+        } else {
+            symbol_choosing();
+        }
+        
         while (!game_ended()) {
             print_board();
             place = ask_move(player);
             add_board_move(player, place);
             change_player();
+            saveGameState();
+        }
+
+    }
+
+    private static void loadGame() {
```

Comment:
Try to add description so it will be more understandable, for example this method for loading the last game so the player can resume the game.

Reply:
Thanks, will do this.